andela/hc-rogue-one
# Ft define team member scope #156577845

#### What does this PR do?
Enable user to define the scope of jobs another user has access to on sending the invite
#### Description of Task to be completed?
GIVEN that a user is inviting another user to have team access to their account
WHEN they are inviting the person via email
THEN they should be able to define the scope of the individual healthchecks jobs for which they have team access
#### How should this be manually tested?
1: Install the project and install the requirements.txt i a virtual environment
2: Set up email configurations from an email server, `host`, `user`, `password`, `port`
3: Change the email backend to support the smtp server you are going to use
4: Create a job/check
5: Invite another user and allow access to the job
6: Integrate the user's email to the job
7: Team user should now receive notifications for that job in case it goes up or down

#### What are the relevant pivotal tracker stories?
#156577845

## Checklist: I completed these to help reviewers




  No? Unsure? -> configure your IDE, format the code and add the changes with `git add . && git commit --amend`

- [] I have **added tests** to cover my changes. (If you refactored
  existing code that was well tested you do not have to add tests)

  No? -> write tests and add them to this commit `git add . && git commit --amend`


  No? -> figure out why and add the fix to your commit. It is your responsibility to make sure your code works.


  No? Unsure? -> execute command `git pull --rebase origin develop`
